docs/.vitepress/zh.ts

```diff
@@ -0,0 +1,60 @@
+import { defineConfig } from 'vitepress'
+
+export const zh = defineConfig({
```

Comment:
Ah this is smart! I wasn’t thinking about all the additional links and navigation items that needed to be translated.